.babelrc

```diff
@@ -1,7 +1,12 @@
 {
   "presets": [
     "react",
-    "env",
+    [
+      "env",
+      {
+        "modules": false
```

Comment:
Can you add comments?

Reply:
This disables  ES module `import` handling by babel (transpilation into `require`) and leaves it to webpack. Webpack uses `import` (especially named `import`) to perform Tree shaking.